src/EditorFeatures/CSharpTest/Diagnostics/Configuration/ConfigureCodeStyle/BooleanCodeStyleOptionConfigurationTests.cs

```diff
@@ -136,7 +136,7 @@ public Customer()
 }
         </Document>
         <AnalyzerConfigDocument FilePath=""z:\\.editorconfig"">[*.cs]    # Comment1
-dotnet_style_object_initializer = false:suggestion    ; Comment2
+dotnet_style_object_initializer = true:suggestion    ; Comment2
```

Comment:
Why was the test passing before this?

Reply:
I think my analysis [below](https://github.com/dotnet/roslyn/pull/38675#discussion_r324007255) was not completely correct. I think previously the analyzer code to fetch option set returned null or default option value as the test diagnostic service does not use WorkspaceAnalyzerOptions. So analyzer flagged the diagnostic. The configuration code fix would still see the AnalyzerConfigDocument as the test workspace does have it, so it will work do the correct update to the editorconfig file.